src/App/App.js

```diff
@@ -30,45 +33,46 @@ function App() {
     fetch(`https://rancid-tomatillos.herokuapp.com/api/v2/movies/${id}`)
     .then((res) => {
       if (!res.ok) {
-        console.log("error");
+        displayError(400);
       } else {
         return res.json();
       } 
     })
     .then((data) => setSelectedMovie(data.movie))
+    .catch(err => displayError(500))
   }
 
-
   function showDesc(evt) {
     getSingleMovie(evt.target.id);
     setIsMovieSelected(true);
   }
+
+  function displayHomePage() {
+    setIsMovieSelected(false)
+    setSelectedMovie({})
+  }
+
+  function displayError(responseLevel) {
+    setIsError(true)
+    setResponseLevel(responseLevel)
+  }
   
   return (
     <React.Fragment>
-      <Nav />
-      <React.Fragment>
-        {!isMovieSelected ? (
-          <Main showDesc={showDesc} movieData={movies} />
-        ) : (
-          <Desc movie={selectedMovie} />
-        )}
-      </React.Fragment>
+      {isError ? (<p className="error">ERROR: {responseLevel} </p>) : (
```

Comment:
I like the way you made this dynamic to show the error code. I might consider adding more messaging for the user. As a user, what should I do if I get an error? Return to the home page? Try again? Leave and come back later?